go.mod

```diff
@@ -5,152 +5,22 @@ go 1.17
 replace (
 	github.com/kudobuilder/kuttl => github.com/layer5io/kuttl v0.4.1-0.20200806180306-b7e46afd657f
 	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
-// golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
+//	golang.org/x/sys => golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8
```

Comment:
Is there a need to keep this commented out? Would be nice to delete it if it's not being used.

Reply:
No good reason, no. If memory serves, I think people were just fearful of losing it in the event a revert was needed.